apps/dotcom-worker/src/lib/routes/createRoomSnapshot.ts

```diff
@@ -18,7 +23,6 @@ export async function createRoomSnapshot(request: IRequest, env: Environment): P
 
 	const persistedRoomSnapshot = {
 		parent_slug: data.parent_slug,
-		slug: roomId,
```

Comment:
This was used for supabase to allow us to query by the slug. It's not needed for R2 though.